app/models/buffer_update.rb

```diff
@@ -1,13 +1,24 @@
 class BufferUpdate < ApplicationRecord
+  SOCIAL_SERVICE = {
```

Comment:
should this be plural?

Reply:
How about `SOCIAL_MEDIA`? My next commit will change it.

Reply:
sure :-)